superset-frontend/src/components/LastUpdated/index.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { useEffect, useState, FunctionComponent } from 'react';
+import moment from 'moment';
+import { t, styled } from '@superset-ui/core';
+import Icon from 'src/components/Icon';
+
+const REFRESH_INTERVAL = 60000; // every minute
+
+interface LastUpdatedProps {
+  updatedAt: string | Date | number;
+  update?: React.MouseEventHandler<SVGSVGElement>;
+}
+moment.updateLocale('en', {
```

Comment:
It looks like the locale is set to `en` in bootstrap data, but I wonder if anyone ever changes that 🤔

Reply:
We do set the locale when apps start up, https://github.com/preset-io/incubator-superset/blob/037c7e24df2cd75ed714f2f56875ce84137ae6fe/superset-frontend/src/preamble.ts#L40

This just changes the way some things are displayed in the english locale, mainly showing time with seconds when formatting timestamps as calendar format. The seconds granularity is desired so refreshes that happen in the same second actually see a change in the UI.